src/main/webapp/app/tutorialgroup/manage/tutorial-free-periods/crud/create-tutorial-group-free-period/create-tutorial-group-free-period.component.html

```
@@ -1,13 +1,5 @@
-<p-dialog
-    [header]="'artemisApp.pages.createOrEditTutorialFreePeriodTitle.create' | artemisTranslate"
-    [modal]="true"
-    [visible]="dialogVisible()"
-    [closable]="true"
-    (visibleChange)="dialogVisible.set($event)"
-    [style]="{ width: '40rem' }"
-    appendTo="body"
->
+<tum-ui-dialog [header]="'artemisApp.pages.createOrEditTutorialFreePeriodTitle.create' | artemisTranslate" size="medium" [(visible)]="dialogVisible">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)